django/db/models/expressions.py

```diff
@@ -1025,7 +1025,13 @@ def __init__(self, queryset, output_field=None, **extra):
 
     def __getstate__(self):
         state = super().__getstate__()
-        state.pop('_constructor_args', None)
+        args, kwargs = state['_constructor_args']
+        if args:
+            queryset = args[0]
+            # Prevent the QuerySet from being evaluated.
+            if queryset is not None:
+                queryset = queryset._chain(_result_cache=[], prefetch_done=True)
```

Comment:
I would if this is something we should do at `__init__` time instead. Wouldn't that allow us to remove the `__getstate__` override entirely?

Reply:
I tried, but without success.